Webapp/sources/templates/auth/register.html

```diff
@@ -57,10 +56,19 @@ <h2>Register</h2><br>
                 </div>
             {% endfor %}
 
+            <div style="margin: auto; width: 60%">
+                {{ form.recaptcha }}
+                {% for error in form.recaptcha.errors %}
+                <div class="alert alert-danger" style="width: 91%" role="alert">
+                    Please confirm you are not a robot!
+                </div>
+                {% endfor %}
+            </div>
+
 
             {{ form.submit() }}
         </form>
     </div>
 </div>
-<input type="checkbox" id="agree-terms" name="agreeterms" hidden>
+<input type="checkbox" id="agree-terms" name="agreeterms" hidden onclick="enablePrivacyCheckbox()">
```

Comment:
What's this onclick for?

Reply:
Just to disable the submit button if it the box is checked. Not sure if it actually works but thought it might be worth a try to add an extra layer of difficulty for the bots